hugegraph-core/src/main/java/org/apache/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -1717,22 +1722,104 @@ private static Query parent(Collection<Query> queries) {
         }
     }
 
+    public static class RemoveLeftIndexJobHelper {
```

Comment:
rename to RemoveLeftIndexQueue?
let RemoveLeftIndexQueue hold a `BlockingQueue<EphemeralJob>`,
and let HugeGraphParams hold a RemoveLeftIndexQueue.
so that we can call `HugeGraphParams graph.submitEphemeralJob(new RemoveLeftIndexJob(xx))`